### ISSUE TITLE
Resize PNG without antialiasing

### ISSUE THREAD

**Issue by mistermiracl** [h2non/imaginary#402]
I have an image that is fully white, meaning it doesn't have any pixels for antialias, so it has no gray border, I would like to keep it that way even after resizing it, is that possible using imaginary?

![antaliased triangle](https://user-images.githubusercontent.com/97761926/191577802-93459840-50cc-4a00-a11c-51010406b972.png)
![pixelated triangle](https://user-images.githubusercontent.com/97761926/191577825-e998920b-b1ed-4735-8cb7-50adc1834739.png)

every pixel in the pixelated triangle is not completely white but it should be its just an example

if you're opening the images in a browser you might need to use `image-rendering: pixelated;` to see what I'm talking about

**Comment by held-m** [comment]
Same problem
![image](https://user-images.githubusercontent.com/24522727/230440610-5e97da0b-8c2c-4f3a-8eae-23d14e76a7f8.png)
![image](https://user-images.githubusercontent.com/24522727/230440736-a4af5281-7c6c-4321-80ff-df057b45d535.png)

**Comment by held-m** [comment]
I figured it out. This is not a bug of imaginary. It's a bug of libvips.

**Comment by mistermiracl** [comment]
> I figured it out. This is not a bug of imaginary. It's a bug of libvips.

You're right, should've updated this issue, here's the discussion on libvips repo, hope its useful!

https://github.com/libvips/libvips/discussions/3083
